client/Components/GetInTouchForm.js

```diff
@@ -11,24 +15,26 @@ const GetInTouchForm = ( { handleFormChange }) => {
 
   }
 
-  const handleFieldBlur = (e) => {
+  const handleFieldBlur = (e, name) => {
 
     const field = e.target.parentElement;
-
+    let newState = {...isHidden};
```

Comment:
I always make objects `const` so I can modify properties, but will be warned if I try to overwrite the entire object